run_pipeline.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# shellcheck disable=SC1143,SC2211,SC2215
+python3 run_pipeline.py \
+#  --input=gs://httparchive/chrome-Jan_1_2022 \
+  --input=gs://httparchive/experimental/input/** \
+#  --input=gs://httparchive/crawls/** \
```

Comment:
TODO: Clean up some of these comments

Reply:
These are all valid pipeline options, however some are only applicable to either streaming or batch dataflow pipelines. I could modify this single bash script to run in either mode given some arguments or I could separate this into two separate scripts (i.e. `run_pipeline_streaming.sh` and `run_pipeline_batch.sh`)

Reply:
Two scripts SGTM